queries/yuck/highlights.scm

```diff
@@ -10,8 +10,6 @@
     (#eq? @keyword.import "include")))
 
 ; Keywords
-; I think there's a bug in tree-sitter the anchor doesn't seem to be working, see
-; https://github.com/tree-sitter/tree-sitter/pull/2107
 (list
   .
   ((symbol) @keyword
```

Comment:
While we're at it, can you also remove the unnecessary `()` at line 15-16

Reply:
or just run `./scripts/format-queries`...

Reply:
> or just run `./scripts/format-queries`...

I did…

Reply:
I haven't add it :)

`grouping` is kindof dangerous is what I think